integ/src/main.rs

```diff
@@ -186,7 +192,12 @@ async fn run() -> Result<()> {
                 .await
                 .context(error::CreatePodSnafu)?;
 
-            // install brupop on EKS cluster
+            // install cert-manager and brupop on EKS cluster
+            info!("Running cert-manager on existing EKS cluster ...");
+            process_cert_manager(Action::Apply, &kube_config_path)
+                .await
+                .context(error::RunBrupopSnafu)?;
+            sleep(WAIT_CERT_MANAGER_COMPLETE);
```

Comment:
```suggestion
            sleep(Duration::from_secs(90));
```

Since this only gets called once and is specific to waiting for `cert-manager` to complete, could we just have the duration set here instead of declaring a constant?

Reply:
I think we should avoid using magical number, name the variable can be more explanatory。

Reply:
Probably doesn't matter but you never know when blocking an async thread could cause a problem. `std::thread::sleep` is blocking, but `toklo::time::sleep` is async.

Reply:
> but toklo::time::sleep is async.

Interesting!